backend/app/core/store.py

```
@@ -122,6 +123,14 @@ def add_reading(device_id: str, sensors: dict, timestamp: datetime | None = None
         sensor_alerts.append(alert)
         new_alerts.append(alert)
 
+    # AI Agent에 센서 데이터 전달 (비동기, 실패해도 무시)
+    try:
+        from app.core.ai_agent import process_sensor_data
+
+        asyncio.ensure_future(process_sensor_data(sensors))
```

Ruff (0.15.9)

[warning] Store a reference to the return value of `asyncio.ensure_future`

(RUF006)